test/propagated-properties.html

```diff
@@ -51,14 +53,18 @@
           placeholder="Pick a date"
           initial-position="1980-01-01"
           show-week-numbers
+          error-message="error-message"
           disabled
+          required
           readonly
         ></vaadin-date-picker>
         <vaadin-time-picker slot="time-picker"
           value="15:00"
           placeholder="Pick a time"
           step="1800"
+          error-message="error-message"
           disabled
+          required
```

Comment:
Here the `error-message` and `required` should be set on the `<vaadin-date-time-picker>` element instead of date picker and time picker, since they should affect the custom field.

This fixture with some properties set directly on date picker and time picker are meant to test the parts where the Flow API may set some properties (when it makes sense) directly on the child elements using the DatePicker and TimePicker API. This basically is testing that properties are synchronized correctly in `__datePickerChanged` and `__timePickerChanged` methods. But not everything can be set correctly via those children like min/max which are special compositions (as time picker min/max is dynamically changed depending on selected date) and label/required/error-message which are mainly for the custom-field.

Reply:
This was done for testing purposes and pushed to remote to save progress (PR is still `in progress`). Removed now.
But anyway thanks for good suggestion!